grafana-dashboards/ambassador.json

```diff
@@ -0,0 +1,2789 @@
+{
```

Comment:
I could go through and review these but that tends to be more painful for anything